tanka-util/util.libsonnet

```diff
@@ -0,0 +1,427 @@
+// Override defaults paramters for objects in the ksonnet libs here.
+local k = import 'k.libsonnet';
+
+k {
+  _config+:: {
+    enable_rbac: true,
+    enable_pod_priorities: false,
+    namespace: error 'Must define a namespace',
+  },
+
+  core+: {
+    v1+: {
+      configMap+: {
+        new(name, data={})::
+          super.new(name, data),
+        withData(data)::
+          if (data == {}) then {}
+          else super.withData(data),
+        withDataMixin(data)::
+          if (data == {}) then {}
+          else super.withDataMixin(data),
+      },
+
+      // Expose containerPort type.
+      containerPort:: $.core.v1.containerPort {
```

Comment:
No need to reference it, +: works

Reply:
Excellent catch, originally this pulled in from `container.portsType`.